numpy/linalg/lapack_lite/clapack_scrub.py

```diff
@@ -240,6 +277,8 @@ def scrubSource(source, nsteps=None, verbose=False):
              ('clean source', cleanSource),
              ('clean comments', cleanComments),
              ('replace dlamch_() calls', replaceDlamch),
+             ('remove definitions', removeSubroutinePrototypes),
+             ('remove builtin functions', removeBuiltinFunctions),
```

Comment:
Prototypes are declarations, not definitions, I think. Both messages should include the word "declaration"